libminifi/test/Utils.h

```diff
@@ -16,8 +16,16 @@
  */
 #pragma once
 
+#ifdef GetObject
+#undef GetObject
+#endif
+
 #include <string>
 #include <utility>
+
+#ifdef GetObject
+#undef GetObject
+#endif
```

Comment:
do we need this twice?

Reply:
Can't remember the reason. I'll remove all of them and add them back one by one on related windows CI failures.